opencti-platform/opencti-front/src/private/components/settings/support/SupportPackages.tsx

```diff
@@ -146,7 +146,8 @@ const SupportPackages = () => {
       <div className="clearfix"/>
       <Paper className="paper-for-grid" variant="outlined" sx={{
         height: '100%',
-        minHeight: '100%',
+        maxHeight: '600px',
+        overflow: 'scroll',
```

Comment:
Scrollbar are displayed all the time, both in X and Y. 
We only need in Y and when the list overflow the max height.
<img width="5108" height="2274" alt="image" src="https://github.com/user-attachments/assets/3dc97962-a8cc-40ca-a02b-97c0950dfa2b" />

This would fix the overflow: 
```suggestion
        overflowY: 'auto',
```